INSTALL.md

```diff
@@ -19,29 +17,54 @@ docker run bystro bystro-annotate.pl #Annotate
 docker run bystro bystro-build.pl #Build
 ```
 
-## Installation on RPM-based distros
+# Direct (non-Docker) installation
 
-###### (Fedora, Redhat, Centos, openSUSE, Mandriva)
+There are 2 components to Bystro:
+ 1. The Bystro annotator: a Perl program accessed through the command line (via bin/bystro-*)
+ 2. The Bystro Python package: where the rest of Bystro's functionality lives (statistics, proteomics, etc).
 
-1.  `git clone https://github.com/akotlar/bystro.git && cd bystro && source ./install-rpm.sh`
+## Installing the Bystro annotator (Perl/cli)
+
+##### (Fedora, Redhat, Centos, openSUSE, Mandriva)
 
-## Installation on MacOS
+1.  `git clone https://github.com/akotlar/bystro.git && cd bystro && source ./install-rpm.sh`
 
-###### (tested on HighSierra, interactive)
+##### MacOS (tested on HighSierra, interactive)
 
 1.  `git clone https://github.com/akotlar/bystro.git && cd bystro && source ./install-mac.sh`
 
-## Installation on Debian systems
+##### Ubuntu
+1.  Ensure that packages are up to date (`sudo apt update`), or that you are satisified with the state of package versions.
+2.  `git clone https://github.com/akotlar/bystro.git && cd bystro && source ./install-apt.sh`
+    - Please note that this installation script requires root priveleges, in order to install system dependencies
```

Comment:
Maybe more direct to say something like:

`Please not that this installation script will ask you for the root password in order to install system dependencies`

if that's what will happen.